### PR TITLE
Checks PricelessPositionManager State phase changes

### PR DESCRIPTION
**Pull request by nicholaspai**
Part of #989

### CODE REVIEW

File: core/test/financial-templates/PricelessPositionManager.js [hunk 1]
```diff
@@ -235,7 +236,8 @@ contract("PricelessPositionManager", function(accounts) {
     assert.equal(sponsorFinalBalance.sub(sponsorInitialBalance).toString(), expectedSponsorCollateral);
     await checkBalances(toBN("0"), toBN("0"));
 
-    // TODO: Add a test to check that normal redemption does not work after maturity.
```
Comment (nicholaspai) [inline core/test/financial-templates/PricelessPositionManager.js]: This is tested in the "Frozen Post Expiry" test

File: core/test/financial-templates/PricelessPositionManager.js [hunk 2]
```diff
@@ -416,9 +421,12 @@ contract("PricelessPositionManager", function(accounts) {
     // Can't transfer if the target already has a pricelessPositionManager.
     await pricelessPositionManager.create({ rawValue: toWei("150") }, { rawValue: toWei("100") }, { from: sponsor });
     assert(await didContractThrow(pricelessPositionManager.transfer(other, { from: sponsor })));
+
+    // Contract state should not have changed.
+    assert.equal(await pricelessPositionManager.contractState(), PositionStatesEnum.OPEN);
   });
 
-  it("Frozen when pre expiry and undercollateralized", async function() {
```
Comment (nicholaspai) [inline core/test/financial-templates/PricelessPositionManager.js]: I think this "pre-expiry" was a typo, and I wasn't sure why "undercollateralized" is important here? Whoever wrote this test, can you clarify?

Comment (nicholaspai) [reply core/test/financial-templates/PricelessPositionManager.js]: @ptare @chrismaree any ideas why we are testing for "undercollateralized" positions here?

Comment (ptare) [reply core/test/financial-templates/PricelessPositionManager.js]: Might have been just copy pasted or maybe the test changed and we didn't update the name.

**Pull request merged**